<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>350</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Update data</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Icons for app Birthday reminder/update.ico</normaloff>../Icons for app Birthday reminder/update.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: #1E1E1E;
}
QPushButton{
	background-color: black;
	color: yellow;
	border: 1px solid #483D8B;
}
QPushButton:hover{
	border: 2px dashed #00FA9A;
	color: #00FA9A;
}
QPushButton:pressed{
	background-color: #2F4F4F;
	color: #FF0000;
	border: 1px dashed #000080;
}
QPushButton:disabled{
	background-color: black;
	color: #800000;
	border: none;
}

QLineEdit{
	background-color: #DCDCDC;
	color: 	#000080;
	border: 1px solid yellow; 
}
QLineEdit:focus{
	border: 3px dashed #800000;
}
QLabel{
	color: #FF4500;
}
QGroupBox{
	backgrond-color: #696969;
	color: #7FFF00;
	border: 1px solid #0000CD;
}
QRadioButton{
	color: #FF4500;
}
QRadioButton:hover{
	color: #00FF00;
}
QTableWidget {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 black, stop:1 blue);
	border: 2px solid #FF4500;
	color: yellow;
}
QTableWidget::item {
    border: 5px solid rgba(68, 119, 170, 150);
    background-color:rgba(68, 119, 170, 125);
}

QHeaderView, QHeaderView::section {
    background-color: rgba(128, 128, 128, 128);
	color: #00CED1;
}
QTableWidget::item:hover {
    border: 2px dashed #00FA9A;
	color: #00FA9A;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>10</y>
      <width>201</width>
      <height>192</height>
     </rect>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <property name="columnCount">
     <number>1</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>200</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>40</number>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Ім'я</string>
     </property>
    </column>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>213</x>
      <y>10</y>
      <width>206</width>
      <height>58</height>
     </rect>
    </property>
    <property name="title">
     <string>Шлях до файлу</string>
    </property>
    <widget class="QLineEdit" name="line_path">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Шлях...</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>80</y>
      <width>226</width>
      <height>136</height>
     </rect>
    </property>
    <property name="title">
     <string>Введення дати народження</string>
    </property>
    <widget class="QLineEdit" name="line_date">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="maxLength">
      <number>5</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>02.12</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_FAQ_date">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_load_data">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Перевірити дату</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_info_names_in_date">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>101</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #00FA9A;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_clear_date">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Очистити поле</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_clear_date_in_data">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Видалити дані</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>220</y>
      <width>417</width>
      <height>127</height>
     </rect>
    </property>
    <property name="title">
     <string>Дії над даними</string>
    </property>
    <widget class="QLineEdit" name="line_name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Ім'я....</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_FAQ_name">
     <property name="geometry">
      <rect>
       <x>195</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_clear_name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Очистити</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_update_in_list">
     <property name="geometry">
      <rect>
       <x>114</x>
       <y>90</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Оновити у списку</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>241</x>
       <y>54</y>
       <width>131</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Додавання даних</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>241</x>
       <y>10</y>
       <width>121</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Заміна даних</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_3">
     <property name="geometry">
      <rect>
       <x>241</x>
       <y>98</y>
       <width>131</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Видалення даних</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_FAQ_replace_data">
     <property name="geometry">
      <rect>
       <x>382</x>
       <y>5</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_FAQ_add_data">
     <property name="geometry">
      <rect>
       <x>382</x>
       <y>50</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_FAQ_remove_data">
     <property name="geometry">
      <rect>
       <x>382</x>
       <y>93</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_exit">
    <property name="geometry">
     <rect>
      <x>533</x>
      <y>315</y>
      <width>55</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Закрити</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_add_in_dict">
    <property name="geometry">
     <rect>
      <x>434</x>
      <y>314</y>
      <width>79</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Зберегти дані</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_info_date_in_file">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>233</y>
      <width>152</width>
      <height>59</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #00FA9A;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_clear_name</sender>
   <signal>clicked()</signal>
   <receiver>line_name</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
